src/components/externalLinks/ExternalLinksCRUD/AddExternalLink.scss

```diff
@@ -104,6 +105,7 @@
                             flex-direction: column;
 
                             input {
+                                width: 604px;
```

Comment:
can't this be derived from the parent div?

Reply:
done